elementary/utils/pydantic_shim.py

```
@@ -11,17 +11,21 @@
     from pydantic import (  # type: ignore  # noqa
         BaseModel,
+        EmailStr,
         Extra,
         Field,
+        ValidationError,
         create_model,
         root_validator,
         validator,
     )
 elif pydantic_major == "2":
     from pydantic.v1 import (  # type: ignore  # noqa
```

Ruff (0.15.0)

[warning] Unused blanket `noqa` directive

Remove unused `noqa` directive

(RUF100)